openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/UpdateDescriptionPage/UpdateDescriptionPage.tsx

```diff
@@ -241,14 +243,7 @@ const UpdateDescription = () => {
                   data-testid="assignees"
                   label={`${t('label.assignee-plural')}:`}
                   name="assignees"
-                  rules={[
-                    {
-                      required: true,
-                      message: t('message.field-text-is-required', {
-                        fieldText: t('label.assignee-plural'),
-                      }),
-                    },
```

Comment:
I think after removing this message will contain "name" instead of translated value!

Reply:
We are passing the validation messages object in the form above. That will ensure that the label will be used in the message.